conf/test_full.config

```diff
@@ -12,13 +12,74 @@ params {
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961089_GSM3415653_MShef11_bulk_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961089_GSM3415653_MShef11_bulk_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ],
+    ],
+    ['SRR7961103_GSM3415667_MShef11_low_oxygen_Q2',
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961103_GSM3415667_MShef11_low_oxygen_Q2_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961103_GSM3415667_MShef11_low_oxygen_Q2_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ],
+    ],
+    ['SRR7961102_GSM3415666_MShef11_low_oxygen_Q1',
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961102_GSM3415666_MShef11_low_oxygen_Q1_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961102_GSM3415666_MShef11_low_oxygen_Q1_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ],
+    ],
+    ['SRR7961104_GSM3415668_MShef11_low_oxygen_Q3',
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961104_GSM3415668_MShef11_low_oxygen_Q3_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961104_GSM3415668_MShef11_low_oxygen_Q3_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ],
+    ],
+    ['SRR7961150_GSM3415714_MShef4_bulk',
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961150_GSM3415714_MShef4_bulk_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961150_GSM3415714_MShef4_bulk_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ],
+    ],
+    ['SRR7961161_GSM3415725_MShef4_J1',
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961161_GSM3415725_MShef4_J1_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961161_GSM3415725_MShef4_J1_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ],
+    ],
+    ['SRR7961164_GSM3415728_MShef4_J3',
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961164_GSM3415728_MShef4_J3_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961164_GSM3415728_MShef4_J3_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ],
+    ],
+    ['SRR7961163_GSM3415727_MShef4_J2',
+      [
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961163_GSM3415727_MShef4_J2_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961163_GSM3415727_MShef4_J2_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ]
+    ]
   ]
-  // Genome references
-  fasta = 'https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa'
-  fasta_index = 'https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa.fai'
+
+  genome = 'GRCh37'
```

Comment:
Is there still no GRCh38?

Reply:
Yeah there is: https://github.com/nf-core/methylseq/blob/4f31ed1792e3cc8ce0ea648b6ee8f541f70102f7/conf/igenomes.config#L26-L37

I think I copied & pasted this from somewhere..

Reply:
```suggestion
  genome = 'GRCh38'
```